### PR TITLE
React and gh pages

### PR DESCRIPTION
**Pull request by victorhuang1201**
Created React app and added material ui dependencies.
Added github pages dependencies so website can be deployed at anytime.

### CODE REVIEW

File: my-app/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "@testing-library/user-event": "^12.8.3",
+    "gh-pages": "^3.1.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
+  "homepage": "https://brianhuang822.github.io/dogTrick/",
+  "scripts": {
+	"predeploy": "npm run build",
```
Comment (brianhuang822) [inline my-app/package.json]: need formatting fix, look into installing eslint and set up some rulesets

File: my-app/src/App.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.Difficulty {
```
Comment (brianhuang822) [inline my-app/src/App.css:1]: with material UI, its better to use the material UI styling. API https://material-ui.com/styles/basics/#higher-order-component-api

File: my-app/src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
```
Comment (brianhuang822) [inline my-app/src/App.test.js:5]: write new tests for your App.js

File: my-app/package.json (hunk 1 above)
Comment (victorhuang1201) [reply my-app/package.json]: I believe I fixed the formatting issue, and I added the some recommended eslint rule sets

File: my-app/src/App.css (hunk 2 above)
Comment (victorhuang1201) [reply my-app/src/App.css:1]: Got a question, I will definitely be using material UI for the components, but do I use it for styling some regular elements such as div or h1 elements?

Comment (brianhuang822) [reply my-app/src/App.css:1]: yes, stick with using material UI styling and remove the css for this project

File: my-app/.eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+	"settings": {
+		"react": {
+			"version": "17.0.2"
```
Comment (brianhuang822) [inline my-app/.eslintrc.js:13]: did you run eslint? this file looks like its indented incorrectly

**Review by brianhuang822 — APPROVED** [review]

File: my-app/.eslintrc.js (hunk 4 above)
Comment (victorhuang1201) [reply my-app/.eslintrc.js:13]: I didn't run it on this file, but I ran it now and no changes were made, I think its fine.

**Pull request merged**